docs/extensions/sandbox/register.md

```diff
@@ -0,0 +1,128 @@
+---
+description: Learn about the ways to register extensions in the sandbox
+contributors: Rijk van Zanten
+---
+
+# Registering Extensions in the Sandbox
+
+The way extensions are registered in the sandbox closely follows the syntax of the existing extension types, but there
+are some subtle yet crucial differences in how data is passed back and forth to the API.
+
+## Hooks
+
+Hooks are registered by exporting a default function that gets an object with the `filter` and `action` properties that
```

Comment:
Do we only get access to the `action` and `filter` events not the `init` and `embed` hook events?

Reply:
That's correct (for now). `init` exposes internals like the express app which kinda negate the point of using a sandbox in the first place; `embed` allows you to add in malicious code to the app head/body, which similarly feels odd in the context of a sandbox